### DISCUSSION TITLE
Search not working on custom asset metadata

### DISCUSSION THREAD

**Discussion by neha04** [pimcore/pimcore#17867] [Q&A / Support]
Hi All,

While searching for asset using custom metadata , no asset is displayed.

**Comment by neha04** [comment]
Hi All,
I have installed enterprise asset metadata bundle.
I have added few localised and few non localised fields.
I am trying to search asset using that metadata but no asset is displayed in quick search.

Is there any configuration that needs to done for quick search or asset metadata search?

**Reply by kingjia90** [reply]
I am not totally sure, but i believe it's not provided out of the box.

For assets, theoretically, it could be extended by simply filling the `properties` column of `search_backend_data` with some conversion of the metadata in text/json(?)
https://github.com/pimcore/pimcore/blob/3ffaeac8d9f07f877ea4f469015be1774de447b9/bundles/SimpleBackendSearchBundle/src/Model/Search/Backend/Data.php#L348
and re-run the search backend reindex command
